foundry.toml

```diff
@@ -50,3 +50,11 @@ runs = 100
 runs = 100
 
 # See more config options https://github.com/foundry-rs/foundry/tree/master/config
+
+
+[profile.default.zksync]
+zksolc = "1.5.1"
+
+
+[etherscan]
+    mainnet = { key = "${ETHERSCAN_API_KEY}" , url = "${ETHERSCAN_API_URL}"}
```

Comment:
This should not be necessary as we explicitly set the env variables in the docs.